src/routes/home/HomeScreen.tsx

```diff
@@ -69,30 +60,40 @@ const HomeScreen = ({ navigation }: HomeScreenProps): JSX.Element => {
   }
 
   const customDisciplineItems = customDisciplines?.map(customDiscipline => (
-    <CustomDisciplineItem
+    <CustomDiscipline
       key={customDiscipline}
       apiKey={customDiscipline}
       navigation={navigation}
       refresh={refreshCustomDisciplines}
     />
   ))
 
-  const disciplineItems = disciplines?.map(item => (
-    <DisciplineListItem key={item.id} item={item} onPress={() => navigateToDiscipline(item)} hasBadge={false} />
+  const selectedProfessionItems = selectedProfessions?.map(profession => (
+    <DisciplineCard
+      key={profession.id}
+      discipline={profession}
+      showProgress
+      onPress={navigateToDiscipline}
+      navigateToNextExercise={navigateToDiscipline} // TODO LUN-290 add progress
+    />
   ))
 
   return (
-    <Root>
-      <Header />
-      <StyledText>{labels.home.welcome}</StyledText>
-      <AddCustomDisciplineContainer onPress={navigateToAddCustomDisciplineScreen}>
-        <AddCircleIcon width={wp('8%')} height={wp('8%')} />
-        <AddCustomDisciplineText>{labels.home.addCustomDiscipline}</AddCustomDisciplineText>
-      </AddCustomDisciplineContainer>
-      {customDisciplineItems}
-      <ServerResponseHandler error={error} loading={loading} refresh={refresh}>
-        {disciplineItems}
-      </ServerResponseHandler>
+    <Root contentContainerStyle={{ flexGrow: 1, justifyContent: 'space-between' }}>
+      <View>
+        <HeaderWithMenu />
+
+        <WelcomeHeading>{labels.home.welcome}</WelcomeHeading>
+        <WelcomeSubHeading>{labels.home.haveFun}</WelcomeSubHeading>
+
+        {selectedProfessionItems}
+
+        {isCustomDisciplineEmpty ? (
+          <AddCustomDisciplineCard navigate={navigateToAddCustomDisciplineScreen} />
+        ) : (
+          customDisciplineItems
+        )}
```

Comment:
There will be a screen for managing them, and there it will be possible. But on the home screen there is just a simple entry for the first one. See designs if interested.